src/cloudsc_python/requirements.txt

```diff
@@ -0,0 +1,7 @@
+click
+gt4py@git+https://github.com/gridtools/gt4py.git#egg=gt4py
+h5py
+numpy
+pandas
+sympl@git+https://github.com/stubbiali/sympl.git@oop#egg=sympl
```

Comment:
I made a number of changes in my fork of Sympl, e.g. adding the ability to disable asserts and checks, and the possibility to feed the model components with the buffers for the output fields, so to avoid any memory allocation inside the components. With the authors of Sympl, we haven't agreed on a strategy to merge these modifications yet. The fact that the discussion includes also my PhD supervisors (and so Thomas Schulthess) makes the process even longer... We have had a paper about this in the pipeline for long time. Adding the CLOUDSC as a use case would surely make the article more appealing, and may push us to finally wrap up the manuscript. But this is for another thread :)

Reply:
Ah, I see. Thanks for the context, good to know.